couchdbkit/resource.py

```diff
@@ -124,6 +129,9 @@ def request(self, method, path=None, payload=None, headers=None, **params):
             else:
                 error = msg
 
+            end_time = datetime.utcnow()
```

Comment:
If `end_time` always gets called in the finally then this looks unnecessary, but I could be misreading the diff